### DISCUSSION TITLE
Genes Removed during FindAlleles

### DISCUSSION THREAD

**Discussion by pursellta** [milaboratory/mixcr#1543] [Ideas]
I am worried about the genes being removed by findalleles. My library only has full VDJRegion coverage of the IgM and only a small portion of the IgG sequences. I had planned use the FindAlleles .json out put as the reference and rerun the analysis with assembling clones by CDR3 (where I have read coverage for my IgG library), but am concerned about V-gene miscalling in the IgGs if there are V-gene found in the IgGs and not IgM (i.e. genes removed from the "custom" referenced generated by FindAlleles).

Any advice on this topic would be much appreciated!

**Comment by mizraelson** [comment]
If you have two .clns files one for IgM and another one for IgG, and both come from the same organism you can use both simultaneously to run findAlleles.

Also, in the latest [develop version](https://link.milaboratory.com/software/mixcr/mixcr-develop.zip) we have the parameter `--dont-remove-unused-genes`

**Reply by pursellta** [reply]
To answer your question, they are in the same .clns; the thing I am concerned about is that the VDJ assemblies for IgG have a gap that includes all of CDR2 and into the surrounding FR regions.

Thanks, I will be using the `--dont-remove-unused-genes` parameter!
